services/core/MasterDriverAgent/master_driver/interfaces/modbus_tk/maps/__init__.py

```diff
@@ -189,23 +210,20 @@ class Map(object):
        all the Fields (Registers) defined in the CSV.
     """
 
-    def __init__(self, file='', map_dir='', addressing='offset',  name='', endian='big', description='', registry_config_lst=''):
-        """
-        :param kwargs: map_dir= ,name=modbus_tk, addressing=offset, endian=big, file=example.csv
-        """
+    def __init__(self, file='', map_dir='', addressing='offset',  name='', endian='big', description='', registry_config_lst='', **kwargs):
```

Comment:
Adding **kwargs without processing it can lead to accidentally covering up spelling mistakes else where in code or in a configuration file. Unless you have a _really_ good reason for this it should probably be removed. What is your intent here?

Reply:
In all of the modbus and modbus_tk tests, for setting up the registers in modbus server, we used Catalog, for example: ModbusClient = Catalog()['modbus_tk_test'].get_class(), to load data from maps.yaml into Map. In maps.yaml, we add an extra field write_multiple_registers for each device type, and use it for generate the driver config by config_cmd tool.

Reply:
If you are using it to consume one extra value for tests I would recommend either calling out the specific fields when calling the constructor (I know, it's verbose and less elegant than calling the constructor with **kwargs) or adding the extra field to either this or a child class and then ignoring it with a comment saying why.

Reply:
I removed **kwargs and just pushed the new change.